Telesign/RestClient.cs

```diff
@@ -132,7 +142,7 @@ public TelesignResponse(HttpResponseMessage response)
         /// <param name="nonce">A unique cryptographic nonce for the request.</param>
         /// <param name="userAgent">User Agent associated with the request.</param>
         /// <returns>A dictionary of HTTP headers to be applied to the request.</returns>
-        public static Dictionary<string, string> GenerateTelesignHeaders(string customerId,
+        public Dictionary<string, string> GenerateTelesignHeaders(string customerId,
```

Comment:
In each of the SDKs the generateTelesignHeaders (or equivalent) function was designed to be static and not require any 3rd party libraries/packages. The reason for this was to allow customers to extract the function implementation "as is" and implement it in their application without having to use the SDK and its dependencies.